docs/RFCs/011-Sdk-Testing-Platform.md

```diff
@@ -0,0 +1,41 @@
+# RFC 011 - Run dotnet test with Microsoft.Testing.Platform
```

Comment:
It seems this was what you worked on in https://github.com/microsoft/testfx/pull/4324. I think it can be deleted for the purpose of this PR.